websites/J/jut.su/dist/metadata.json

```diff
@@ -0,0 +1,21 @@
+{
+  "author": {
+    "name": "Yokkkoso && Baka",
+    "id": "190762645316567040"
+  },
+  "service": "jut.su",
+  "description": {
+    "en": "Jut.su is Russian anime website",
+    "ru": "Jut.su это русский аниме сайт"
+  },
+  "url": "jut.su",
+  "version": "1.0.0",
+  "logo": "https://i.imgur.com/uijmPIc.png",
+  "thumbnail": "https://i.imgur.com/NlBgvR0.png",
```

Comment:
try getting a better quality logo, and the logo doesn't seem to be the same one as featured on their website.

Reply:
I use them flaticon